setup.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from os import sys
+
+try:
+    from skbuild import setup
+except ImportError:
+    print('scikit-build is required to build from source.', file=sys.stderr)
+    print('Please run:', file=sys.stderr)
+    print('', file=sys.stderr)
+    print('  python -m pip install scikit-build')
+    sys.exit(1)
+
+setup(
+    name='itk-stlmeshio',
+    version='1.0.0',
+    author='Luis Ibáñez',
+    author_email='community@itk.org',
+    packages=['itk'],
+    package_dir={'itk': 'itk'},
+    download_url=r'https://github.com/InsightSoftwareConsortium/ITKSTLMeshIO',
+    description=r'MeshIO class to read and write ITK meshes into STL files.',
+    long_description='itk-stlmeshio provides a clas to read and write ITK '
+                     'meshes into STL files.\n'
+                     'Please refer to:\n'
+                     'Ibáñez L. "STL file format MeshIO class for ITK", '
```

Comment:
\n

Reply:
In reality, the `\n` is not producing the expected new line if one has a look at the PyPI documentation... So I've just left the ones that have been used throughout the rest of the remotes. In this case, the name of the IJ article and the rest of the publication information are not separated by a "new line" commonly in the rest of the remotes.